tests/testthat/helper.R

```diff
@@ -58,8 +56,8 @@ mod.mlr <- mlr::train(lrn, task)
 predictor.mlr <- Predictor$new(mod.mlr, data = iris)
 
 ## mlr3
-task_iris <- TaskClassif$new(id = "iris", backend = iris, target = "Species")
-learner_iris <- lrn("classif.rpart", predict_type = "prob")
+task_iris <- mlr3::TaskClassif$new(id = "iris", backend = iris, target = "Species")
+learner_iris <- mlr3::lrn("classif.rpart", predict_type = "prob")
```

Comment:
This does not help much here as we need the mlr3 namespace for the `$train()` calls.